nessie/jobs/import_calnet_data.py

```diff
@@ -53,15 +53,21 @@ def _put_calnet_data_to_s3(sids):
         missing = set(sids) - set(ldap_sids)
         app.logger.warning(f'Looked for {len(sids)} SIDs but only found {len(all_attributes)} : missing {missing}')
 
-    students = []
-    for a in all_attributes:
+    serialized_data = ''
+    total_count = len(all_attributes)
+    for index, a in enumerate(all_attributes):
+        sid = a['csid']
+        app.logger.info(f'CalNet import: Fetch attributs of student {sid} ({index + 1} of {total_count})')
         sortable_name = a['sortable_name'] if 'sortable_name' in a else ''
         name_split = sortable_name.split(',')
         full_name = [name.strip() for name in reversed(name_split)]
-        students.append({
-            'sid': a['csid'],
-            'uid': a['uid'],
+        serialized_data += json.dumps({
+            'sid': sid,
+            'affiliations': ','.join(a['affiliations']),
```

Comment:
@johncrossman It looks like this line is doing something funny when there's only one affiliation type and affiliations is a string rather than an array. In the dev schema I see affiliations `F,O,R,M,E,R,-,S,T,U,D,E,N,T` and `S,T,U,D,E,N,T,-,T,Y,P,E,-,R,E,G,I,S,T,E,R,E,D`.

Reply:
Nice catch, @paulkerschen  Strange things are afoot in CalNet data types